web/package.json

```diff
@@ -4,7 +4,8 @@
   "description": "Terminal sharing server with web interface - supports macOS, Linux, and headless environments",
   "main": "dist/server/server.js",
   "bin": {
-    "vibetunnel": "./bin/vibetunnel"
+    "vibetunnel": "./bin/vibetunnel",
+    "vt": "./bin/vt"
```

Comment:
We can't put this here, this would break installing the package if a different vt is installed, and there are quite a few!